db/write_thread.cc

```diff
@@ -269,8 +269,8 @@ void WriteThread::CompleteFollower(Writer* w, WriteGroup& write_group) {
   SetState(w, STATE_COMPLETED);
 }
 
+static WriteThread::AdaptationContext jbg_ctx("JoinBatchGroup");
```

Comment:
ah, static global variable is local to the current file, is that correct? If so the current approach is good.